api/python/properties_prediction_d/config.py

```diff
@@ -0,0 +1,24 @@
+"""Config file"""
+
+
+# Dataset
+
+file_name = "Adrenergic_dataset.csv"
+target = "logP"  # "AdrA1A_PCHEMBL_VALUE"
+smiles = "Structure"
```

Comment:
I think, this should not be predefined. Better to set it as input parameters in the notebook or CLI.